src/internal/views/nunjucks/charge-information/workflow-tabs/to-set-up.njk

```diff
@@ -20,7 +20,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for row in licences.toSetUp %}
+                {% for row in licences.toSetUp.data %}
                     <tr class="govuk-table__row">
                         <td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>
```

Comment:
Please consider wrapping `<td class="govuk-table__cell">{{row.licence.licenceNumber}}</td>` with a href to the licence ID

Reply:
will run this past UCD maybe and add it to another ticket